File: iOS/JapaneseBenkyo/Struct/Kanji.swift
//
//  Kanji.swift
//  JapaneseBenkyo
//
//  Created by 김호성 on 2/14/24.
//

import Foundation

struct OldKanji: Codable, Equatable, Hashable {
    let kanji: String
    let hanja: String
    let eumhun: String
    let jpSound: String
    let jpMeaning: String
    let examples: [String]
}

struct Kanji: Codable, Equatable, Hashable {
    let kanji: String
    let hanja: String
    let eumhun: String
    let jpSound: String
    let jpMeaning: String
    let examples: [Vocabulary]
}

File: Android/app/src/main/java/com/constant/japaneseeveryday/model/Kanji.kt
package com.constant.japaneseeveryday.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
class Kanji(
    val kanji: String,
    val hanja: String,
    val eumhun: String,
    val jpSound: String,
    val jpMeaning: String,
    val examples: List<Vocabulary>
) : Parcelable {

}